packages/common/test/customChains.spec.ts

```diff
@@ -63,12 +61,12 @@ describe('[Common]: Custom chains', () => {
 
   it('custom() -> behavior', () => {
     let common = createCustomCommon({ chainId: 123 })
-    assert.deepEqual(common.networkId(), BigInt(1), 'should default to mainnet base chain')
+    assert.equal(common.consensusAlgorithm(), 'casper', 'should default to mainnet base chain')
```

Comment:
Is this right? Why are we not checking chainID?

Reply:
Yes, that's correct, the goal of this test is to *not* check for the value passed (`chainId`), but to check that the other values remain the mainnet-specific ones.